### DISCUSSION TITLE
added a /furnace command

### DISCUSSION THREAD

**Discussion by FrozerYTB** [EssentialsX/Essentials#5358] [Ideas and Feature Suggestions]
add a /furnace command that allows to cook our ores in our hand

**Comment by mdcfe** [comment]
Unlike the existing workbench commands, furnaces require an actual block entity to keep track of the cooking time. We would need to simulate the furnace block entity somehow for this to work.

**Reply by FrozerYTB** [reply]
it's just an order to process minerals.

For example, if we have iron ore, we use /furnace to transform it into iron ingots in an instant.

**Reply by Chew** [reply]
So wouldn't /smelt be more appropriate?

**Reply by FrozerYTB** [reply]
if we can say both
